06.bowling_object/game.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require_relative 'frame'
+
+class Game
+  def initialize(scores)
+    score_index = 0
+    @frames = []
+    (1..10).each do |frame|
+      if frame == 10
+        @frames << Frame.new(frame, scores[score_index], scores[score_index + 1], scores[score_index + 2])
+      elsif scores[score_index] == 'X'
+        @frames << Frame.new(frame, 'X', 0)
```

Comment:
```suggestion
        @frames << Frame.new(frame, 'X')
```

Gameクラスが気を利かせて0を加えるよりも、Frameオブジェクト自身が'X'を適切に取り扱える方がオブジェクト指向らしいと思います。